recipe/meta.yaml

```diff
@@ -1,34 +1,38 @@
-{% set version = "1.14.1" %}
+{% set version = "1.15.0" %}
 
+# Update the numpy value here to the most recent released version
+# when updating to a new scipy feature release
 {% if numpy is not defined %}
-{% set numpy = "2.0.0" %}
+{% set numpy = "2.2.1" %}
 {% endif %}
 {% set numpy_minor = numpy.split(".")[1] | int %}
 
+# ensure is_freethreading gets detected as a used variable
+# [is_freethreading]
+
 package:
   name: scipy-split
   version: {{ version }}
 
 source:
-  # The sdist distributed by scipy contains pre-compiled pythran sources,
```

Comment:
Note that this was not true since the switch to Meson. We should probably switch back to sdist from PyPI, so we can avoid all this manual git submodule handling?

Reply:
It's a bit annoying to update the hashes, but I do prefer building from source. I really don't want to pick up anything pre-cythonized or otherwise precompiled. If that's not the case anymore, I'm happy to switch back to the upstream tarballs.

Reply:
There are no pre-generated sources except for the trivial capturing of the git hash that's being built from in `version.py` (that one is necessary because it's VCS info). And that one is actually broken in the current recipe, switching to a proper sdist will fix it:
```python
>>> scipy.version.git_revision
'Unknown'
``` 

The GitHub-generated tarballs should never be used for Python packages if it can be avoided, because they're broken in multiple ways - this missing versioning info will happen for many Python packages.